appinfo/info.xml

```diff
@@ -5,6 +5,10 @@
 	<summary>A face recognition app</summary>
 	<description><![CDATA[**Detect and group faces of your loved one in your cloud**
 
+⚠️ This application requires minimum of 12GB of RAM memory to work! See [Requirements](https://github.com/matiasdelellis/facerecognition/wiki/Requirements-and-Limitations) for details.
```

Comment:
12GB????
I have only 8.. :disappointed:

Reply:
OMG, good you noticed:) I didn't copy "-" nicely:)